packages/lib/rateLimit.ts

```diff
@@ -0,0 +1,27 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import cache from "memory-cache";
+
+import { ErrorCode } from "./auth";
+
+const rateLimit = (options: { interval: number }) => {
+  return {
+    check: (limit: number, token: unknown) => {
```

Comment:
```suggestion
    check: (limit: number, token: string) => {
```

Correct me if I'm wrong @agustif